src/Adapter/TaxRulesGroup/CommandHandler/BulkDeleteTaxRuleHandler.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
```

Comment:
Warning: your license files are wrong 😄 see https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/project-modules/

`2007-2019 PrestaShop and Contributors`
⬇️ 
`2007-2019 PrestaShop SA and Contributors`